bundles/org.eclipse.swt/Eclipse SWT/win32/org/eclipse/swt/graphics/Image.java

```diff
@@ -1338,6 +1392,9 @@ public ImageData getImageData (int zoom) {
 	} else if (imageFileNameProvider != null) {
 		ElementAtZoom<String> fileName = DPIUtil.validateAndGetImagePathAtZoom (imageFileNameProvider, zoom);
 		return DPIUtil.autoScaleImageData (device, new ImageData (fileName.element()), zoom, fileName.zoom());
+	}
+	else if (this.dataAtBaseZoomFactor != null) {
```

Comment:
nitpicky: formatting issue
```suggestion
	} else if (this.dataAtBaseZoomFactor != null) {
```